code/popup.html

```diff
@@ -11,80 +11,272 @@
   <link rel="shortcut icon" type="image/png" href="/icon/icon.png"/>
   <title>OpenGenus: World's first Offline Search Engine</title>
   <meta charset='utf-8'>
+   <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+   <style type="text/css">
+
+#searchSuggest .ui-menu-item{
+    list-style: none outside none;
+    width: 100%;
+
+    padding: 0px;
+
+
+   }
+
+    #search {
+    margin-bottom: 15px;
+}
+#searchSuggest {
+    width: 100%;
+border:3px solid black;
+
+
+    margin-bottom: 15px;
+}
+
+#searchSuggest .ui-autocomplete {
+  width: 100%;
+    position: relative;
+    float: none;
+}
+#searchSuggest .ui-menu-item:hover{
+  background-color: #D3D3D3;
+}
+#searchSuggest .ui-menu .ui-menu-item {
+    float: none;
+
+}
+
+
+
+   </style>
+  <link rel="stylesheet" href="/resources/demos/style.css">
+  <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+  <script>
+
+  $( function() {
+
+
+
+
+var availableTags = [
```

Comment:
For the tags, you must use `tags.json` that is available at [Cosmos Search](https://github.com/OpenGenus/cosmos-search).